src/classes/Addresses.cls

```diff
@@ -302,16 +300,20 @@ public inherited sharing class Addresses extends fflib_SObjects {
                     NPSP_Contact npspContact = new NPSP_Contact(con);
                     // detect that the contact's current address was deleted.
                     // and if so clear any override flag so it will get the default address.
+                    if (npspContact.hasAddressOverride() == false) {
```

Comment:
Does it make sense to check this after the `npspContact.currentAddress() == null` check below it, which would change the override value?

Reply:
yeah good call. updated!